packages/dd-trace/src/config.js

```diff
@@ -480,6 +480,7 @@ ken|consumer_?(?:id|key|secret)|sign(?:ed|ature)?|auth(?:entication|orization)?)
     this.appsec = {
       enabled: DD_APPSEC_ENABLED,
       rules: DD_APPSEC_RULES ? safeJsonParse(maybeFile(DD_APPSEC_RULES)) : require('./appsec/recommended.json'),
+      customRulesProvided: DD_APPSEC_RULES !== undefined,
```

Comment:
It could also be null tho

Reply:
I cannot see how. 

Value is coalesced [here](https://github.com/DataDog/dd-trace-js/blob/master/packages/dd-trace/src/config.js#L302) and the output will be a string or undefined, but maybe I'm missing something?

Reply:
What I'm saying is this line should be consistent with line 482 above, both should make the same check, otherwise it's inconsistent and may lead to stupid edge cases